### PR TITLE
PP-7585 Add integration tests for payment types

### PR DESCRIPTION
**Pull request by kbottla**
## WHAT
- Added missing tests for payment types (get/update) scenarios

### CODE REVIEW

File: test/integration/payment-types/payment-types.it.test.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+      return whenGetPaymentTypes(app).expect(200)
+    })
+    it('should return error for get payment types when connector errors', async () => {
+      connectorMock.get(CONNECTOR_ACCOUNT_CARD_TYPES_PATH)
+        .reply(500)
+
+      return whenGetPaymentTypes(app)
+        .expect(500)
+        .expect(response => expect(response.text).to.contain('Unable to fetch payment types'))
```
Comment (adityapahuja) [inline test/integration/payment-types/payment-types.it.test.js:74]: Do we need to include "Please try again or contact support team."?

```suggestion
        .expect(response => expect(response.text).to.contain('Unable to fetch payment types. Please try again or contact support team.'))
```

**Review by adityapahuja — APPROVED** [review]
The changes look good to me. My suggestion is optional and it shouldn't block you from merging the pull request.

File: test/integration/payment-types/payment-types.it.test.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      connectorMock.post(CONNECTOR_ACCOUNT_CARD_TYPES_PATH, { card_types: ['mastercard-id-1234'] })
+        .reply(200)
+
+      return whenPaymentTypesUpdated(app, { debit: 'mastercard-id-1234' })
+        .expect(302)
+    })
+    it('should redirect when no card is selected to update', async () => {
+      return whenPaymentTypesUpdated(app, {})
+        .expect(302)
+        .expect(response => console.log(response.text))
```
Comment (alexbishop1) [inline test/integration/payment-types/payment-types.it.test.js:88]: Sure we want to do this logging?

Comment (adityapahuja) [inline test/integration/payment-types/payment-types.it.test.js:88]: Do we need to validate response.text?

```suggestion
        .expect(response => expect(response.text).to.contains('Found. Redirecting to /account/account-external-id/payment-types'))
```

Comment (kbottla) [reply test/integration/payment-types/payment-types.it.test.js:88]: 🤦🏽 I thought I removed this but clearly not.

Comment (kbottla) [reply test/integration/payment-types/payment-types.it.test.js]: 👍🏽 Updated!

File: test/integration/payment-types/payment-types.it.test.js (hunk 1 above)
Comment (kbottla) [reply test/integration/payment-types/payment-types.it.test.js]: 👍🏽 Updated!

**Review by adityapahuja — APPROVED** [review]
These changes look good to me.

**Pull request merged**